README.md

```diff
@@ -79,23 +79,23 @@ directory.
 
 ### resources/
 
-If the package provides a root-level directory for other resource files, it MUST
+If the package provides a root-level directory for other resource files (such as language files), it MUST
```

Comment:
It does also include things like images, CSS, and Javascript doesn't it?

Reply:
It depends on whether you're using an asset pipeline or not. It becomes too opinionated from this point, so I omitted those, since I believe that this project is not about the content of the folders.

Reply:
Oh sure, that makes sense. Thanks.